javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,77 @@
+// AGE-IFY exercise
+
+const yearOfBirth = 1985;
+const yearfuture = 2045;
+
+const myAge = yearfuture - yearOfBirth;
+console.log(myAge);
+
+console.log("You will be " + myAge + " " + "years old in " + yearfuture + ".");
+
+
+// Goodbye OLdboy exercise 
+const dogYearOfBirth = 2018; 
+const dogYearFuture = 2025;
+ const dogYear = dogYearFuture - dogYearOfBirth ;
+ const dogInHumanYear = dogYear * 7 ;
```

Comment:
I think it is opposite 
```suggestion
 const dogYear = (dogYearFuture - dogYearOfBirth) * 7 ;
 const dogInHumanYear = dogYear / 7 ;
```